lib/broadside/cli/gli_commands.rb

```diff
@@ -0,0 +1,142 @@
+def add_tag_flag(cmd)
```

Comment:
it does get us a significant step closer to not having to have everything in the `Deploy` object which is great. maybe once things calm down we can revisit if it makes any sense but in the midst of a big refactor is prolly not the time